v_3_3_4/master_template.go

```diff
@@ -2020,7 +2020,7 @@ coreos:
       [Service]
       TimeoutStartSec=300
       Restart=always
-      RestartSec=0
+      RestartSec=2
```

Comment:
Is that change related?

Reply:
Change is not related, it was just something Timo suggested, anyway I will move this to another PR. It was there just for some tests.